tip-wlan/charts/wlan-ssc-service/templates/deployment.yaml

```diff
@@ -216,7 +216,7 @@ spec:
             - name: {{ .Values.service.name3 }}
               containerPort: {{ .Values.service.port3 }}
               protocol: TCP
-            {{- end }}              
+            {{- end }}
```

Comment:
Why is this file changed?

Reply:
cleaned up a bunch of whitespaces at the end